### ISSUE TITLE
EPIC: Accessibility WCAG 2.1

### ISSUE THREAD

**Issue by rschwietzke** [Xceptance/posters-demo-store#157]
# Goal
Enable the storefront to be partially complained to [WCAG 2.1 ](https://www.w3.org/TR/WCAG21/) guidelines. This is for testing and demo purposes only. It should demonstrate how this works, is implemented as well as tested.

# Non-Goals
The backoffice does not have to be updated. It is also not needed to really fulfill all rules and features. 

# Requirements
Deliver all basic rules except for rules that require video, audio, or external device support.
